iocore/net/UnixNet.cc

```diff
@@ -465,6 +433,9 @@ NetHandler::mainNetEvent(int event, Event *e)
     epd = (EventIO *)get_ev_data(pd, x);
     if (epd->type == EVENTIO_READWRITE_VC) {
       vc = epd->data.vc;
+      if (cop_list.in(vc)) {
+        cop_list.remove(vc);
+      }
```

Comment:
Step 1. put all vc in open_list into cop_list.
Step 2. remove vc from cop_list if it has READ/WRITE event. (running 100 times)
Step 3. call check_inactivity() to processing cop_list.
Step 4. go to step 1.